python-modules/README.rst

```diff
@@ -0,0 +1,11 @@
+Python Modules Links
```

Comment:
It's not fully clear to me how this is working. I understand how the links are being created in `extract-translation-source-files`, and I understand that we need to be able to look things up by module name instead of repo name, but it's not clear to me how this works on the `edx-platform` side.

Is the plan to update `edx-platform` to look specifically for a `python-modules` directory as part of the sparse checkout via atlas?

Reply:
@brian-smith-tcril yes, this has been tested locally with the edx-platform.

The following is the management command that is responsible about pulling the XBlock translations by module name:

 - https://github.com/Zeit-Labs/edx-platform/pull/2/files#diff-69c7c1e7f1ec08e9d4f9a4a00fae62392ceaecb644f8ad697888e3eefc4a3bc2R27-R45

We'll share a demo soon to clarify the design we're targeting, but for this PR I think it's ready and it's ready to merge after once you find that the documentation is clear for the purpose of this repository.

Reply:
I think the biggest thing I've having trouble with is the name of the directory. If we look at the root of `openedx-translations` now, it's fairly clear what every directory is for:
```
docs
requirements
scripts
translations
```
Each directory name clearly implies what the directory is for and what it does. It is not clear from reading `python-modules` what that directory is for, what it does, and why it lives in the root of the repo.

Reply:
yes, sure. `openedx-translations` is a good name. What about `edx-platform-translations`? It sounds even more specific imho.

Reply:
> translations/edx-platform-links/

That should be less confusing. Thanks!